wasp/main.py

```diff
@@ -3,4 +3,6 @@
 
 import wasp
 from gadgetbridge import *
+
+wasp.system.set_theme(b'\xef{\xef{\xef{\xef{\x00\xf8\xe0\x07<\xe7iJiJ\xef{\xb6\xb5\xb6\xbd\xff\xff')
```

Comment:
Also test code... should not be committed.

Reply:
same as in `wasp/boards/simulator/main.py`

Reply:
In the next commit it's removed.